readme.md

```diff
@@ -1022,11 +1022,14 @@ For example, it can be helpful during development to avoid an infinite number of
 ###### pagination.stackAllItems
 
 Type: `boolean`\
-Default: `true`
+Default: `false`
+
+Defines how the property `allItems` in [`pagination.paginate`](#pagination.paginate), [`pagination.filter`](#pagination.filter) and [`pagination.shouldContinue`](#pagination.shouldContinue) is managed.
+
+By default, the property `allItems` is always an empty array. This setting can be helpful to save on memory usage when working with a large dataset.
```

Comment:
The second sentence makes less sense now that it's `false` by default. It should be formulated differently.

Reply:
Done